etc/update-index.py

```diff
@@ -0,0 +1,17 @@
+import os
```

Comment:
I really loathe bash scripting (and I can't write it either) so I just made another python script.